core/modules/commands/savewikifolder.js

```diff
@@ -176,7 +176,8 @@ WikiFolderMaker.prototype.saveCustomPlugin = function(pluginTiddler) {
 	this.saveJSONFile(directory + path.sep + "plugin.info",pluginInfo);
 	self.log("Writing " + directory + path.sep + "plugin.info: " + JSON.stringify(pluginInfo,null,$tw.config.preferences.jsonSpaces));
 	var pluginTiddlers = $tw.utils.parseJSONSafe(pluginTiddler.fields.text).tiddlers; // A hashmap of tiddlers in the plugin
-	$tw.utils.each(pluginTiddlers,function(tiddler) {
+	$tw.utils.each(pluginTiddlers,function(tiddler,title) {
+		if(!tiddler.title) tiddler.title = title;
```

Comment:
It's a minor coding style thing, but TiddlyWiki always uses braces in "if" statements:

```
if(!tiddler.title) {
   tiddler.title = title;
}
```